### ISSUE TITLE
OWallet 0.9.2 Recovering problem

### ISSUE THREAD

**Issue by makrusw2** [ontio/OWallet#117]
Good day! Today I just updated the OWallet from 0.9.1 to 0.9.2 and tryied to import my wallet ( I created new wallet through OWallet on 0.8.9 version), I have set keystore.db path and tryied to recover by that file. Unfortunately, I met a permanently loading. How I can solve this problem?

**Comment by arnosk** [comment]
I have the same problem with 0.9.3. After update the individual wallet don't appear. And importing doesn't work. Keeps on loading foreever

**Comment by MickWang** [comment]
@makrusw2 Hello, What file did you try to import for recover? Did you try the v0.9.3?
@arnosk  The owallet does not remind you to set the path of keystore.db file at first?

**Comment by arnosk** [comment]
Yes it does, and I set it to the path with the existing keystore.db from a previous version. And that clearly doesnot work

**Comment by MickWang** [comment]
If your PC is Windows, Please try run it as Administrator.
